### ISSUE TITLE
Error Validation Message Not Working

### ISSUE THREAD

**Issue by UjalaJha** [imrealashu/laravel-multiauth#8]
The Registration and Login Page shows no error on wrong validation, instead gets reloaded.

**Comment by imrealashu** [comment]
Yes. I have wrote that long time ago didn't notice. Thanks for letting me know I'll fix this at the earliest or I'd love to receive a PR.

**Comment by UjalaJha** [comment]
I have solved this error,
Actually All routes in laravel 5.2 by defualt includes web middleware, if
you check php artisan route:list for you, it will show 2 web middleware,
simply remove the route file one and boom, it works :)
